packages/cli/test/commands/unpack.test.js

```diff
@@ -6,23 +6,15 @@ import { stubCommands, itShouldParse } from './share';
 describe('unpack command', function() {
   stubCommands();
 
-  itShouldParse(
-    'should call unpack script with a name',
-    'unpack',
-    'zos unpack ZepKit',
-    function(unpack) {
-      unpack.should.have.been.calledWithExactly({ repoOrName: 'ZepKit' });
-    },
-  );
+  itShouldParse('should call unpack script with a name', 'unpack', 'zos unpack ZepKit', function(unpack) {
```

Comment:
Should we also change all these `ZepKit`/`zepkit` to `starter`? I know it won't change the functionality, but it will help removing all deprecated references.